cmd/evm/runner.go

```diff
@@ -62,7 +62,18 @@ func runCmd(ctx *cli.Context) error {
 		ret  []byte
 		err  error
 	)
-	if ctx.GlobalString(CodeFlag.Name) != "" {
+	if fn := ctx.Args().First(); len(fn) > 0 {
```

Comment:
What do you mean which version? Are there more versions? I'm a little confused...